SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -1743,6 +1743,26 @@ void USpatialReceiver::ResolveIncomingOperations(UObject* Object, const FUnrealO
 
 		USpatialActorChannel* DependentChannel = ChannelObjectPair.Key.Get();
 		UObject* ReplicatingObject = ChannelObjectPair.Value.Get();
+		
+		// Check whether the resolved object has been torn off, or is on an actor that has been torn off.
+		if (AActor* AsActor = Cast<AActor>(ReplicatingObject))
+		{
+			if (AsActor->GetTearOff())
+			{
+				UE_LOG(LogSpatialActorChannel, Log, TEXT("Actor to be resolved was torn off, so ignoring incoming operations. Object ref: %s, resolved object: %s"), *ObjectRef.ToString(), *Object->GetName());
+				UnresolvedRefsMap.Remove(ChannelObjectPair);
+				continue;
+			}
+		}
+		if (UActorComponent* AsActorComponent = Cast<UActorComponent>(ReplicatingObject))
```

Comment:
I think we should be getting `GetTypedOuter<AActor>(ReplicatingObject)`, since `ReplicatingObject` doesn't in fact have to be an actor component for this to happen.

Reply:
Sounds good, will do 👍